server/src/main/java/org/elasticsearch/index/mapper/IgnoredFieldMapper.java

```diff
@@ -40,14 +40,14 @@ public static class Defaults {
         }
     }
 
+    public static final IgnoredFieldType FIELD_TYPE = new IgnoredFieldType();
+
```

Comment:
This makes things a bit easier to test, in a roundabout way - if we have direct public access to this FieldType then we don't need a SearchExecutionContext to build an empty StoredFieldsPhase, as the default metadata fields can be added to the list of fields to process directly.